drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/importer.rb

```diff
@@ -480,6 +481,7 @@ def involved_project_ids
                     data_source_id: incoming.data_source_id,
                     PersonalID: incoming.PersonalID,
                   ).with_deleted.update_all(incoming.slice(klass.upsert_column_names(version: '2020')))
+                  @updated_source_client_ids << incoming.id
```

Comment:
You won't have .id, but you will have a single data_source_id and PersonalIDs, so store the PersonalID for later.